examples/combobox/snapshot/base-open-class-name.jsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-console, react/prop-types */
+import React from 'react';
+import Combobox from '~/components/combobox/combobox';
+import Icon from '~/components/icon';
+import escapeRegExp from 'lodash.escaperegexp';
+import IconSettings from '~/components/icon-settings';
+
+const accounts = [
+	{ id: '1', label: 'Acme', subTitle: 'Account • San Francisco', type: 'account' },
+	{ id: '2', label: 'Salesforce.com, Inc.', subTitle: 'Account • San Francisco', type: 'account' }
+];
+
+const accountsWithIcon = accounts.map((elem) => Object.assign(elem, {
+	icon: <Icon
+		assistiveText="Account"
+		category="standard"
+		name={elem.type}
+	/> })
+);
+
+class Example extends React.Component {
+	constructor (props) {
+		super(props);
+
+		this.state = {
+			inputValue: '',
+			selection: []
+		};
+	}
+
+	render () {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<Combobox
+					className="this-is-the-input"
+					classNameContainer="this-is-the-container"
```

Comment:
hm, so I think in other components we use `containerClassName` (ie. app launcher, icon, selected-listbox). Can we change this and the other to `menuClassName`?

Reply:
I'm worried about discoverability when they are alphabetical. What do you think about changing the other ones instead?